geom/src/polygon.rs

```diff
@@ -587,8 +568,12 @@ impl Polygon {
         Ok(results)
     }
 
+    /// If simplification fails, just keep the original polygon
```

Comment:
This is an example where all the callers of this would end up doing this kind of thing anyway